src/classes/BDI_DataImportFLSService.cls

```diff
@@ -115,7 +115,11 @@ public class BDI_DataImportFLSService {
 
                 for (SObjectType objectType : targetFieldsByObject.keySet()) {
                     for (SObjectField field : targetFieldsByObject.get(objectType)) {
-                        if (!isValid(field)) {
+                        //Flag to allow for FLS check bypass for CampaignMember.CampaignId
+                        Boolean campaignMemberByPass =
+                                UTIL_Describe.getObjectDescribe(objectType).getName().equals('CampaignMember') &&
+                                        field.getDescribe().getName().equals('CampaignId');
```

Comment:
What do you think about updating the comparison here to use strongly typed tokens instead of Strings? Like
`if (objectType == CampaignMember.SObjectType
                                && field == CampaignMember.CampaignId) {`
And, see comment below about how this comparison could be done inside a new `isValidateUpdateException` method instead of here.